packages/snap/src/index.ts

```diff
@@ -1,13 +1,13 @@
 import {buildBaseFiles} from './generate-snapshots/build-base-files';
 import {generateSnapshots} from './generate-snapshots/generate-snapshots';
 import {runEyes} from './snap-test/snap';
-import { createTempDirectory } from 'create-temp-directory';
+import {createTempDirectory} from 'create-temp-directory';
 import Registry from '@ui-autotools/registry';
 
-export async function eyesTest(projectPath: string) {
+export async function eyesTest(projectPath: string, fs: any) {
```

Comment:
We could define our own interface that contains only relevant functions. Something like:
```
interface IFilesystem {
  writeFileSync: (file: string, data: string) => void;
  ...
}
```